swagger.yaml

```diff
@@ -3788,6 +4038,67 @@ components:
           description: ID of last modifier
           readOnly: true
 
+    Template:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        data:
+          $ref: "#/components/schemas/TemplateData"
+        visibility:
+          $ref: "#/components/schemas/TemplateVisibility"
+        createdAt:
+          type: string
+          format: date-time
+          description: Created date
+          readOnly: true
+        modifiedAt:
+          type: string
+          format: date-time
+          description: Modified data
+          readOnly: true
+        creatorId:
+          type: string
+          format: uuid
+          description: ID of creator
+          readOnly: true
+        lastModifierId:
+          type: string
+          format: uuid
+          description: ID of last modifier
+          readOnly: true
+
+    TemplateData:
+      type: object
+      properties:
+        exportThemeId:
+          type: string
+          format: uuid
+        allowAnonymous:
+          type: boolean
+          description: Are anonymous replies allowed or not
+        title:
+          type: string
+        sections:
+          type: array
+          items:
+            $ref: "#/components/schemas/MetaformSection"
+        defaultPermissionGroups:
```

Comment:
Is it actually possible to include defaultPermissionGroups into templates? I believe that permission groups are form-specific, so it would not make any sense to include them into templates but I'm not absolutely certain about this.